static_src/stores/service_plan_store.js

```diff
@@ -62,6 +62,27 @@ class ServicePlanStore extends BaseStore {
         break;
       }
 
+      case serviceActionTypes.SERVICE_INSTANCES_RECEIVED: {
+        const instances = this.formatSplitResponse(action.serviceInstances);
+        this.fetched = false;
+        this.fetching = true;
+        this.emitChange();
+        const planRequests = [];
+        for (const instance of instances) {
+          planRequests.push(cfApi.fetchServicePlan(instance.service_plan_guid));
```

Comment:
That's a good point. Most of our logic now just fetches the entities and doesn't check that they're there. But in reality, these plans aren't changing so it's pretty safe to check.

Should we set a pattern to not fetch by default and apply it here?